services/common/src/components/forms/EditForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from "react";
+import { Form } from "antd";
+import { compose } from "@reduxjs/toolkit";
+import { connect, useDispatch, useSelector } from "react-redux";
+import {
+    ConfigProps,
+    reduxForm,
+    submit,
+    getFormSubmitErrors,
+    InjectedFormProps,
+} from "@mds/common/components/forms/form";
+
+interface EditFormProps {
+    children: any;
+    layout?: "inline" | "horizontal" | "vertical";
+    name: string;
+    initialValues?: any;
+    onSubmit?: (values) => void | Promise<void>;
+    reduxFormConfig?: Partial<ConfigProps>;
+    formClassName: string;
+}
+
+const EditForm: FC<EditFormProps & InjectedFormProps<any>> = ({
+    children,
+    layout,
+    name,
+    initialValues,
+    onSubmit,
+    formClassName,
+}) => {
+
+    const dispatch = useDispatch();
```

Comment:
really not significantly different than existing FormWrapper, extracted this out from there